src/lime/tools/AndroidHelper.hx

```diff
@@ -395,6 +395,7 @@ class AndroidHelper
 
 	public static function uninstall(packageName:String, deviceID:String = null):Void
 	{
+		return;
```

Comment:
You can't just disable an AndroidHelper function because you want Air apps to function differently. Find the call to `uninstall()` and disable that.